src/style.css

```
@@ -0,0 +1,42 @@
+.container {
+  height: auto;
+  width: auto;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-of-type(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
```

no need to repeat it, just use a comma
```suggestion
.stars--1 .stars__star:nth-of-type(1),
.stars--2 .stars__star:nth-child(-n+2) {
  background-image: url(./images/star-active.svg);
}
```